pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -406,8 +410,10 @@ def decomposition(wires):
         ]
         return decomp_ops
 
-    def adjoint(self):
-        return SX(wires=self.wires).inv()
+    def adjoint(self, do_queue=False):
```

Comment:
I wonder if we should just get rid of the `do_queue` keyword in the abstract method, as no child operation seem to either accept the `do_queue` keyword or do anything with it. Shouldn't break anything if we just get rid of it.